com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/validation/EclSyntaxErrorMessageProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.b2international.snomed.ecl.validation;
```

Comment:
Please add the standard copyright header to the top, and it should be good to go!

Reply:
Done